events/tests/test_event_images_v1.py

```diff
@@ -217,6 +217,76 @@ def test__upload_an_image_with_api_key(
     assert image.format == "PNG"
 
 
+@pytest.mark.django_db
+def test__image_edit_as_superuser(
+    api_client, list_url, image_data, user, organization, organization2, user2
+):
+    organization.admin_users.add(user)
+    api_client.force_authenticate(user)
+
+    response = api_client.post(list_url, image_data)
+
+    user2.is_superuser = True
+    organization2.admin_users.add(user2)
+    api_client.force_authenticate(user2)
+    detail_url = reverse("image-detail", kwargs={"pk": response.data["id"]})
+    response2 = api_client.put(detail_url, {"name": "this is needed"})
+    assert response2.status_code == 200
+
+
+@pytest.mark.django_db
+def test__image_edit_as_external(api_client, external_user):
+    uploaded_image = create_uploaded_image()
+    image = Image.objects.create(image=uploaded_image, created_by=external_user)
+
+    external_user.is_external = True
```

Comment:
Not needed. `external_user` is already external.

Reply:
`is_external` is a property which checks if the user belongs to any groups.

Reply:
Removed.